new-pipeline/tutorial.html

```diff
@@ -13,21 +13,61 @@
     <link rel="stylesheet" href="style/jquery.tocify.css">
     <style>
     #tocify-header0 > .tocify-item { font-weight: bold; } /* bold the top-level entries */
+    body {
+        padding: 0 5px 5px 5px;
+    }
+    .active {
+        background: rgb(241, 236, 236);
+        border-left: 5px solid #286090;
+    }
+    hr {
+        margin: 0 auto;
+    }
+    nav a{
+        color:black;
+    }
+    nav .nav-stacked li a{
+        color:rgb(85, 81, 81);
+    }
+    #side-nav {
+        padding-top: 2.5em;
+    }
+    .btn-group {
+        padding: 15px;
+    }
+    @media only screen and (max-width: 992px) {
+    #side-nav {
+        display: none;
+    }
+}
     </style>
     <script async src="https://www.googletagmanager.com/gtag/js?id=UA-49796218-2"></script>
 </head>
 <body>
-    <form id="histogram-filters" class="navbar-form inline" style="position: fixed; right: 1em; top: 1em; z-index: 1">
-        <a href="./dist.html" class="btn btn-default" id="switch-views-histogram" title="Switch to the histogram view">Histogram View</a>
-        <a href="./evo.html" class="btn btn-default" id="switch-views-evolution" title="Switch to the evolution view">Evolution View</a>
-        <a href="./" class="btn btn-default btn-primary" title="Landing page for Telemetry dashboards">Dashboards Home</a>
-    </form>
-    <div class="container-fuild" style="margin-top: 1em">
+    <nav class="navbar navbar-default" style="background:rgba(255, 255, 255, 0); border-color: rgba(255, 255, 255, 0); ">
```

Comment:
Instead of specifying `style` attribute, let's move this to the CSS section above.
There are a few more cases like this below that we should change.